osi_object.proto

```diff
@@ -62,6 +62,10 @@ message Vehicle
     // of the front axle under neutral load conditions. In object coordinates.
     optional Vector3d bbcenter_to_front = 11;
     
+    // The flag indicates that if the vehicle is moving. In SensorData, if the vehicle has the same velocity as the ego vehicle, it is hard to 
```

Comment:
Besides the switch to the logic needed (i.e. we need is_stationary instead of is_moving so that the default value is less dangerous), while trying to add this field, I stumbled upon the comment that in SensorData the relative velocity makes detection of a stationary vehicle more problematic. However in SensorData, no Vehicles are present, they are all in GroundTruth only.  Was sensor-relative ground truth meant with this comment? I've separated this change out to a new separate PR #121, since it is unrelated to the Ego-Vehicle duplication code...